### DISCUSSION TITLE
HTTPS Everywhere extension (vs) HTTPS - Only mode in firefox

### DISCUSSION THREAD

**Discussion by tiwarys** [EFForg/https-everywhere#19894] [Q&A] [answered]
Are there any advantages of using HTTPS Everywhere extension over HTTPS only mode in firefox?

**Comment by cschanaj** [comment, marked as answer]
See https://www.eff.org/https-everywhere/faq#https-only-mode

**Comment by zoracon** [comment]
Currently working on logic that will ease any conflict this extension and the HTTPS Only mode will have programmatically
